.gitignore

```diff
@@ -119,9 +119,6 @@ temp/
 tmp/
 face_detection_masking/tmp/
 .vscode/
-.idea/
-google-cloud-sdk/
-
```

Comment:
why do we remove these?

Reply:
It's weird, I didn't remove any of them.